### DISCUSSION TITLE
"Used by counter" for non-master branch?

### DISCUSSION THREAD

**Discussion by FezVrasta** [community/community#23278] [New to GitHub]
<p>Hi, my project (<a href="https://github.com/popperjs/popper-core" rel="nofollow noopener">https://github.com/popperjs/popper-core</a>) used to be <code>popper.js</code> on npm for several years. Recently, I released a major version where it has been renamed to <code>@popperjs/core</code> on npm.</p>
<p>When I cut the new release, I moved the old code into a different branch (v1.x) but since then, the “Used by counter” is only showing the new package name and not the old one.</p>
<p>Is there any way to get back the old package?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/fezvrasta">@fezvrasta</a>,</p>
<p>Thank you for being here! Give it a look when you have a moment, I’ve made the association for you. It should work in a few, and start showing the correct used by counter on your main repo.</p>

**Comment by FezVrasta** [comment, marked as answer]
<p>Thanks! It’s there again now</p>
